### DISCUSSION TITLE
New plugins: "plugin request" label only to be assigned if approved

### DISCUSSION THREAD

**Discussion by mkbloke** [streamlink/streamlink#3586] [General]
I've kind of brought up aspects of this in other issues previously.

Given that there are only mostly 2-3 devs active that assign labels and determine what may or may not get accepted, can it be made a common practice that the "plugin request" label or something else is applied after it's approved as something that will be accepted into master?

**Comment by gravyboat** [comment]
Sure I just added one.

**Reply by mkbloke** [reply]
Excellent, thank you!
